ansible/examples/slave.yml

```diff
@@ -194,28 +193,23 @@
     - name: ensure that 127.0.1.1 is present with an actual hostname
       sudo: true
       lineinfile:
+        backup: yes
         dest: /etc/hosts
         line: '127.0.1.1 ceph-builders'
 
-    # we need to update the cloud templates because 'manage_etc_hosts' is
-    # set to true on the image we use in OVH and some jobs will reboot
-    # these nodes causing the /etc/hosts file to be replace with these
-    # templates making jobs fail because the hostname is not resolvable
-    # not all our images have this setting though, so ignore failures on
-    # those nodes
-    - name: update the etc cloud templates for debian /etc/hosts
+    - name: update etc cloud templates for debian /etc/hosts
       sudo: true
       lineinfile:
+        backup: yes
         dest: /etc/cloud/templates/hosts.debian.tmpl
         line: '127.0.1.1 ceph-builders'
-      failed_when: false
 
-    - name: update the etc cloud templates for debian /etc/hosts
+    - name: update etc cloud templates for red hat /etc/hosts
       sudo: true
       lineinfile:
+        backup: yes
         dest: /etc/cloud/templates/hosts.redhat.tmpl
         line: '127.0.1.1 ceph-builders'
-      failed_when: false
```

Comment:
Why did you remove ``failed_when: false`` from these last two tasks? Those template files won't exist on all slaves and I think ``lineinfile`` will fail if the ``dest`` doesn't exist.